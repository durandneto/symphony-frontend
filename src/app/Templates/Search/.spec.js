import React from 'react'
import renderer from 'react-test-renderer'
import Template from '.'

const templateObject = {
  placeholder: "find a package",
  value: "react",
}

describe('Search Template', () => {
  it ('Rendering Search template without crashing', () => {
    const result = renderer.create(
      <Template
        {...templateObject}

        onEvent={e => {
          console.log(e)
          action(e)
        }}
      />
    ).toJSON()
    expect(result).toMatchSnapshot()
  })
})
