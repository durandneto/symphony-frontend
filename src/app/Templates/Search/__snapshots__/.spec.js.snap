// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Template Rendering Search template without crashing 1`] = `
<div
  className="sc-AxjAm hTSXUB"
>
  <div
    className="sc-AxjAm sc-AxirZ fssdFE"
  >
    <div
      className="sc-AxjAm sc-AxiKw jRLkDN"
    >
      <img
        alt="Symphony search"
        src="Symphony-logo.jpg"
        title="Symphony search"
      />
    </div>
  </div>
  <div
    className="sc-AxjAm sc-AxirZ fssdFE"
  >
    <div
      className="sc-AxjAm sc-AxiKw jRLkDN"
    >
      <input
        className="sc-AxheI gUIYOU"
        disabled={false}
        loading={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyUp={[Function]}
        placeholder="find a package"
        type="text"
        value="react"
      />
    </div>
  </div>
</div>
`;
